pkg/controllers/job/plugins/factory.go

```diff
@@ -20,7 +20,7 @@ import (
 	"sync"
 
 	"volcano.sh/volcano/pkg/controllers/job/plugins/env"
-	"volcano.sh/volcano/pkg/controllers/job/plugins/interface"
+	pluginsinterface "volcano.sh/volcano/pkg/controllers/job/plugins/interface"
```

Comment:
If alias, is there any related code modification?

Reply:
the package name is `pluginsinterface` and the last part of import path `volcano.sh/volcano/pkg/controllers/job/plugins/interface` is `interface`, not equal to package name `pluginsinterface`. So we should use alias. If not, goimports will failed with information File is not `goimports`-ed with -local volcano.sh (goimports)